src/include/duckdb/storage/serialization/nodes.json

```diff
@@ -543,90 +543,86 @@
         "type": "bool"
       },
       {"id": 114,
-        "name": "sample_chunk_size",
+        "name": "sample_size_chunks",
         "type": "idx_t"
       },
       {"id": 115,
-        "name": "sample_chunks",
```

Comment:
Removing any option from the serialization or changing field ids of existing options is a non-backwards compatible change that breaks the storage version. We don't fully support deleting fields yet (CC @Maxxen) - but the correct way of deleting a field is to leave it in the JSON and label it as deleted (e.g. adding `"deleted": true`) so the system can then insert a `Skip` for the field in the deserialization code (in case an old version is read).

For now this doesn't matter that much (yet) but it will matter in a few weeks :)

Reply:
That makes a lot of sense, would you like me to add the deleted as a good practice?

Reply:
We don’t actually support it yet, so we can leave it as is for now

Reply:
Ah sorry, I missed that in your previous comment.